web/packages/teleterm/src/ui/services/workspacesService/documentsService/types.ts

```diff
@@ -133,6 +133,14 @@ export interface DocumentGatewayCliClient extends DocumentBase {
   status: '' | 'connecting' | 'connected' | 'error';
 }
 
+export interface DocumentGatewayKube extends DocumentBase {
+  kind: 'doc.gateway_kube';
+  rootClusterId: string;
+  leafClusterId: string | undefined;
```

Comment:
You don't need these two fields for this document type AFAIK.

Reply:
I got an error here when removing there 😃 
https://github.com/gravitational/teleport/blob/e444901d32e76ccbb1e3c37d8aaa85eb2fca9c22/web/packages/teleterm/src/ui/DocumentTerminal/useDocumentTerminal.ts#L219-L238

Reply:
Ah, I forgot about that. 🤦 Let's just add the same comment above them to say they're tech debt, `DocumentGatewayCliClient` already has a comment like that.